Purchases/Networking/Operations/GetSubscriberDataOperation.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  GetSubscriberDataOperation.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class GetSubscriberDataOperation: CacheableNetworkOperation {
```

Comment:
wondering if this should still be `GetCustomerInfo`, and we keep the customerinfo naming as far as we can, until the request path is defined?

Reply:
Yeah, good point.

Reply:
Updated!